src/main/java/carpet/script/CarpetExpression.java

```diff
@@ -99,11 +92,7 @@
 import net.minecraft.structure.StructurePiece;
 import net.minecraft.structure.StructureStart;
 import net.minecraft.text.LiteralText;
-import net.minecraft.util.ActionResult;
```

Comment:
Yeah, fix those, and change in your IDE not to do that again.

Reply:
how to fix the IDE tho?